### ISSUE TITLE
ERROR Message: cURL error: [60] Peer certificate cannot be authenticated with known CA certificates

### ISSUE THREAD

**Issue by pablorsk** [Quixotix/PHP-PayPal-IPN#39]
Days ago, script return this error message:
ERROR Message: cURL error: [60] Peer certificate cannot be authenticated with known CA certificates

It's something about api_cert_chain.crt or is about my server?

I have the same script in two domains and both began to return the same error.

Regards...

**Comment by petert82** [comment]
Yes, it's most likely the cert chain file causing this. We ran into the same thing, and replaced the existing `api_cert_chain.crt` with `cacert.pem` from [here](http://curl.haxx.se/docs/caextract.html) and it appears to resolve this (obviously assuming you update `ipnlistener.php` to point to the new file, too).

As it stands I'd recommend moving away from relying on this project directly, as it appears to be no longer maintained. We started our [own fork](https://github.com/toolani/paypal-ipn-verifier-php) as a more-or-less drop in replacement, but I believe there are others too that are maintaining working versions of the project.
